docs/rules/order.md

```diff
@@ -94,6 +94,31 @@ You can set the options like this:
 "import/order": ["error", {"groups": ["index", "sibling", "parent", "internal", "external", "builtin"]}]
 ```
 
+### `pathGroups: [array of objects]`:
+
+To be able so group by paths mostly needed with aliases pathGroups can be defined.
+
+Properties of the objects
+
+| property | required | type    | description   |
+|----------|:--------:|---------|---------------|  
+| pattern  |     x    | string  | minimatch pattern for the paths to be in this group (will not be used for builtins or externals) |
+| group    |     x    | string  | one of the allowed groups, the pathGroup will be positioned relative to this group |
+| position |          | integer | defines where around the group the pathGroup will be positioned <br> negative -> before, 0 = like the group, positive -> after <br> default: 1 <br> maximal magnitude: 1000 |
```

Comment:
i think "like the group" should be the default; and i think that the alternative should be "before" or "after", and ordering in the config array itself should determine relative priority.